private-templates-service/adaptivecards-templating-client/swagger.yaml

```diff
@@ -517,18 +505,8 @@ definitions:
   TemplatePreviewUser: 
     type: object
     properties:
-      firstName: 
-        type: string
-      lastName: 
+      authId: 
         type: string
-      team: 
-        type: array
-        items: 
-          type: string
-      org: 
-        type: array
-        items: 
-          type: string
   TemplateInfo: 
     type: object
     properties:
```

Comment:
Can remove the PostedUser definition and post user endpoint as well!

Reply:
has this been resolved?